app/controllers/users_controller.rb

```diff
@@ -7,4 +7,10 @@ def show
     @user = User.find(params[:id])
     @recent_posts = @user.recent_posts
   end
+
+  # private
+
+  # def user_params
+  #   params.require(:user).permit(:name, :email, :password, :password_confirmation)
+  # end
```

Comment:
- [ ] Kindly remove all the commented code to make the codebase look clean.

Reply:
- Well resolved 👍🏽